Neos.ContentGraph.DoctrineDbalAdapter/src/DoctrineDbalContentGraphSchemaBuilder.php

```diff
@@ -29,27 +33,19 @@ public function buildSchema(): Schema
 
     private function createNodeTable(Schema $schema): void
     {
-
         $table = $schema->createTable($this->tableNamePrefix . '_node');
-        $table->addColumn('relationanchorpoint', Types::STRING)
-            ->setLength(255)
-            ->setNotnull(true);
-        $table->addColumn('nodeaggregateid', Types::STRING)
-            ->setLength(64)
-            ->setNotnull(false);
-        $table->addColumn('origindimensionspacepoint', Types::TEXT)
-            ->setNotnull(false);
-        $table->addColumn('origindimensionspacepointhash', Types::STRING)
-            ->setLength(255)
-            ->setNotnull(false);
-        $table->addColumn('nodetypename', Types::STRING)
-            ->setLength(255)
-            ->setNotnull(true);
-        $table->addColumn('properties', Types::TEXT) // TODO longtext?
-        ->setNotnull(true);
+        $table = DbalSchemaFactory::addColumnForNodeAnchorPoint($table, 'relationanchorpoint');
+        $table = DbalSchemaFactory::addColumnForNodeAggregateId($table, 'nodeaggregateid', false);
+        $table = DbalSchemaFactory::addColumnForDimensionSpacePoint($table, 'origindimensionspacepoint', false);
+        $table = DbalSchemaFactory::addColumnForDimensionSpacePointHash($table, 'origindimensionspacepointhash', false);
+        $table = DbalSchemaFactory::addColumnForNodeTypeName($table, 'nodetypename');
+        $table->addColumn('properties', Types::TEXT)
```

Comment:
There is not, but doctrine just creates a LONGTEXT if the length isn't limited as in this case, I first added a "this will result in a longtext as we do not restrict length" comment but found it silly because after this review no one will probably know why that comment is there in the first place.

Reply:
Yes, totally agree, thanks for checking!